examples/server_profile.rb

```diff
@@ -15,10 +15,52 @@
   password: ENV['ONEVIEWSDK_PASSWORD']
 }
 
+# my_server_profile_template = 'spt1'
+# my_server_hardware_type = 'SY 480 Gen9 2'
+# my_server_hardware = '0000A66101, bay 5'
+# my_enclosure_group = 'eg1'
 my_server_profile_template = 'spt1'
-my_server_hardware_type = 'SY 480 Gen9 2'
-my_server_hardware = '0000A66101, bay 5'
-my_enclosure_group = 'eg1'
+my_server_hardware_type = 'BL660c Gen9 1'
+my_server_hardware = 'Encl1, bay 2'
+my_enclosure_group = 'EG_1'
+
+# use this to create server profile with volume attachments using api200 or api300
```

Comment:
Perhaps these data samples could be loaded from a different file, and we could have a comment here for instructing the user on that?
It feels like a lot of visual noise from VAs data when I want to see a server profile example.

Reply:
I'm thinking move all examples of volume attachments to another file, not just the data samples, what do you think?

Reply:
Thinking better, I'll just put the variables within separate file, because looks more simple, and it avoids create similar files, with similar examples and code of variables duplicated.